react-app/src/components/dashboard.jsx

```diff
@@ -7,19 +8,33 @@ import Leaderboard from "./dashboard/leaderboard";
 import BalanceActions from "./dashboard/balanceActions";
 
 class Tokens extends Component {
+  state = {
+    first: "",
+    balance: ""
+  };
+
+  async componentWillMount() {
+    const { data } = await http.get("http://localhost:3900/api/users/me");
```

Comment:
add a todo, the URL is hardcoded, so will break if we start splitting out the services to docker containers.